torchtext/experimental/models/roberta_model.py

```diff
@@ -0,0 +1,74 @@
+import os
+import torch
+import torch.nn as nn
+from torchtext.experimental.modules import TransformerEncoder
+from .roberta_transform import RobertaTransform
+from torchtext.utils import download_from_url, extract_archive, files_exist, load_args_from_json
+
+
+class RobertaModel(nn.Module):
+    """Contain a Roberta encoder model."""
+
+    def __init__(self, args, encoder):
+        super(RobertaModel, self).__init__()
+        self.args = args
+        self.encoder = encoder
+        self.transform = None
+
+    @classmethod
+    def build_model(self, args):
+        encoder = TransformerEncoder(args.ntoken, args.embed_dim, args.nhead,
+                                     args.feedforward_dim, args.nlayers, args.dropout)
+        return cls(args, encoder)
+
+    def load_transform(self, directory='./', tokenizer_file="sentencepiece.bpe.model", vocab_file='vocab.txt'):
+        self.transform = RobertaTransform.from_pretrained(directory=directory,
+                                                          tokenizer_file=tokenizer_file,
+                                                          vocab_file=vocab_file)
```

Comment:
Why have a method for this, instead of just passing the `transform` as an argument to the constructor?